### DISCUSSION TITLE
Embedded text cuts off last 3 characters, did i misconfigure something?

### DISCUSSION THREAD

**Discussion by pehmee** [jakobdylanc/llmcord.py#79] [General]
Hello, for starters; _thank you_ for making such a nice and simple tool.

I installed the bot succesfully.  For some reason i'm seeing Bot's responses (short, long) to be cut off prematurely by 3 chars at least, maybe 4-5. (kinda funny when the bot replies to you with "How may i ass"

Then switched plain_responses to true and the messages won't be cut from the end anymore. I'm using openrouter, and just tried one model, so i cannot say if cuts because of model, but the **plain response is working correctly**, i'd assume the culprit is found somewhere in embedded messages.

I didn't want to open a issue about this whilst i'm not sure if its due to me misconfigurating the bot. Before even looking into it more deeply, i though it would be good idea to just ask about this in discussion.

Anyhoo, great stuff @jakobdylanc 💯

**Comment by jakobdylanc** [comment]
Thanks for the kind words! :)

Are you using the latest version of llmcord? The issue you're describing sounds like one I fixed a few weeks ago.
You can make sure llmcord is up-to-date by running: `git fetch && git reset --hard origin/main`

What specific OpenRouter model did you see this issue with?

**Reply by pehmee** [reply]
It was a fresh install with git clone. Running it with docker. I used liquid/lfm-40b:free

**Reply by jakobdylanc** [reply]
I see the issue on my end as well. Thanks for bringing this to my attention.

I investigated and this is likely the underlying provider (Lambda)'s fault for implementing streamed responses improperly. There's an issue with the format of the received chunks.

For now you'll have to either switch to a different provider or keep `use_plain_responses` enabled. I'll see if I can contact Lambda / OpenRouter support to get this fixed on their end.

**Comment by jakobdylanc** [comment]
This issue is now fixed in latest llmcord. (4e24cdd)
